packages/store/README.md

```diff
@@ -49,9 +49,4 @@ wm.set(h, 'new-data');
 map.delete(h);
 ```
 
-Note that when you import and use the `makeExternalStore` function, the platform
```

Comment:
@michaelfig , as a drive by, I just added a commit for removing this paragraph. It should no longer be here, right?

Reply:
You should go further and delete the entire `# External Store` section... there is no `makeExternalStore` exported from `@agoric/store` at all.

Reply:
Done